### PR TITLE
Pools data list

### PR DESCRIPTION
**Pull request by codewiththeflowxyz**
### Helipad
image

### Mobile version
image

### Uniswap Info
image

Calculations are based on: 

https://github.com/Uniswap/v3-info
https://github.com/Uniswap/v3-info/blob/master/src/data/pools/poolData.ts#L217

### CODE REVIEW

**Review by iceanddust — APPROVED** [review]
LGTM, the only comment I have is that maybe the pool fee should have a different background color other than the primary so it's not confused with a main interactable item. Maybe a grayed-out version of the main green. Wdyt?

[LGTM](https://lgtm.lol/i/809)

**Review by iceanddust — APPROVED** [review]

File: packages/hooks/src/pools/usePoolsData.ts [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+};
+
+export const usePoolsData = (addresses: string[] = [], allowedTokens: string[]): PoolsHookResponse => {
+  const {
+    data,
+    error,
+    refetch: mutate,
+    isInitialLoading
+  } = useQuery<Pool[]>({
+    queryKey: ['pools', URL_UNISWAP_SUBGRAPH_MAINNET, addresses],
```
Comment (bitsforpeace) [inline packages/hooks/src/pools/usePoolsData.ts:103]: Does this not work on goerli? Wondering why mainnet is hardcoded.

**Review by bitsforpeace — APPROVED** [review]
Nice work! Cool to see this data. Just one question about mainnet vs goerli. And also maybe on mobile we can hide everything except the TVL column:
Screenshot 2023-08-25 at 9 25 43 AM

**Comment by codewiththeflowxyz** [comment]
> Nice work! Cool to see this data. Just one question about mainnet vs goerli. And also maybe on mobile we can hide everything except the TVL column: Screenshot 2023-08-25 at 9 25 43 AM

Good point, I'll work on a mobile version.

File: packages/hooks/src/pools/usePoolsData.ts (hunk 1 above)
Comment (codewiththeflowxyz) [reply packages/hooks/src/pools/usePoolsData.ts:103]: At first when I worked on this I didn't see any subgraph for goerli. I'll take a look again.

**Comment by codewiththeflowxyz** [comment]
Mobile version
image

File: packages/hooks/src/pools/usePoolsData.ts [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+import { Pool, PoolsHookResponse } from './pools';
+import { TRUST_LEVELS, TrustLevelEnum } from '../constants';
+import request, { gql } from 'graphql-request';
+import { useDeltaTimestamps } from '@jetstreamgg/utils';
+import { useBlocksFromTimestamps } from '../blocks/useBlocksFromTimestamps';
+import { useMemo } from 'react';
+import { useChainId } from '../network/useChainId';
+import { getUniswapSubgraphUrl } from '../helpers/getSubgraphUrl';
+
+const get2DayChange = (valueNow: string, value24HoursAgo: string): number => {
```
Comment (inclinedmind) [inline packages/hooks/src/pools/usePoolsData.ts:11]: should this be 1 day, not 2 day?

**Pull request merged**